README.md

```diff
@@ -153,7 +153,7 @@ new DefaultPlugin({
 
 If you do not, events are still stored in the `DefaultData.eventLogs`.
 
-When you create your plugin, you can send event to Leto-modelizer to see the progression of your parsing or rendering action.
+When you create your plugin, you can send events to Leto-modelizer to see the progression of your parsing or rendering action.
```

Comment:
Missing changelog update and migration guides

Reply:
done